core/src/main/kotlin/group/Handler.kt

```diff
@@ -224,7 +224,7 @@ class GroupHandler(override val kodein: Kodein) : KodeinAware {
                 logger.error(e) { "Error registering agent $agentId" }
             }
             async(handler) {
-                agentManager.register(info.id, regInfo.copy(name = agentId, description = agentId))
+                agentManager.register(info.id, regInfo.copy(name = agentId, description = agentId, environment = ""))
```

Comment:
What is the value in regInfo.environment, why do we change it? As I see we have a default value of environment = ""

Reply:
regInfo. environment-contains the value set for the service group and in the current implementation it will be set for all agents of this group , you are right here you need to put not an empty string, but a string from the agent config